build.py

```diff
@@ -304,13 +299,13 @@ def releaseId() :
 # Here we're actually doing the build, this will run either locally or inside
 # the container bootstrapped above
 
-if os.path.exists( "/.dockerenv" ) and args.project == "gaffer" :
+if os.path.exists( "/run/.containerenv" ) and args.project == "gaffer" :
 
 	# Start an X server so we can generate screenshots when the
 	# documentation builds.
-	os.system( "Xvfb :99 -screen 0 1280x1024x24 &" )
```

Comment:
I'd suspect the addition of the `.0` to the display number here to be redundant as the screen number is already being specified by `-screen 0`. Did you find this necessary?

Reply:
You're completely right. I was floundering around trying to get the docs build to work, and matched this to the CI in the hope that was the problem. But it wasn't - it was the `.dockerenv` vs `.containerenv` difference that made the difference. And I was too lazy to revert the X changes and wait for another build to see if they were also necessary or superfluous. I've done that test now, and reverted the unnecessary changes in the squashed https://github.com/GafferHQ/build/pull/43/commits/e3a7cf03c5bf3785845d334cd00ef6a6463f7b0a. Thanks for pointing that out.